openslides/mediafiles/views.py

```diff
@@ -15,6 +15,13 @@ class MediafileViewSet(ModelViewSet):
     access_permissions = MediafileAccessPermissions()
     queryset = Mediafile.objects.all()
 
+    def get_queryset(self):
+        user = self.request.user
+        if user.has_perm('mediafiles.can_see_private'):
+            return Mediafile.objects.all()
```

Comment:
Please use something like `queryset = super().get_queryset()` so that line 16 is still in effect.